cobalt/build/configs/linux-x64x11/args.gn

```diff
@@ -11,3 +11,7 @@ enable_vulkan = false
 # Overriding flag from //media/media_options.gni.
 # Cobalt doesn't use //third_party/ffmpeg.
 media_use_ffmpeg = false
+
+# Overriding flag from components/nacl/features.gni
+# NACL platform is deprecated in 2022 : https://sites.google.com/a/google.com/sandboxing-mitigations/reviews/nacl
+enable_nacl = false
```

Comment:
veriied that by default it was set to true for linux builds: https://paste.googleplex.com/5001849489588224 by printing out the value